frontend_app_workspace_advanced/src/component/SpaceSubscriptionsRequests.jsx

```diff
@@ -0,0 +1,93 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import { translate } from 'react-i18next'
+import {
+  DropdownMenu,
+  formatAbsoluteDate,
+  NoHoverListItem,
+  UserInfo
+} from 'tracim_frontend_lib'
+
+export const SpaceSubscriptionsRequests = props => {
+  return (
+    props.subscriptionsRequests.length
+      ? (
+        <span className='workspace_advanced__subscriptionsRequest'>
```

Comment:
```suggestion
        <span className='workspace_advanced__subscriptionRequests'>
```
(and other occurrences if you decide to apply this suggestion)

Reply:
Updated